charters/PM/charter.md

```diff
@@ -0,0 +1,60 @@
+# PM Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the DCP *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
+
+## Definitions
+
+* A **Thematic Roadmap** identifies and expresses customer needs as a small collection of themes or strategic objectives which:
+    * Focuses the DCP community on a documented set of high-level customer priorities
+    * Excites users about the future of DCP
+
+* **Request for Comments (RFC)** is the transparent process that manages all significant enhancements to the DCP software technical and governance documentation, both normative and informative
+
+## In-scope
+
+*Governance* 
+
+* Develop standard processes and best practices for DCP community governance including (but not limited to):
+    * Program and technical oversight, decision making, and conflict resolution
+    * Charters for DCP projects
+    * Enhancements (RFC) to DCP software or governance 
+    * Planning
+    * Communication
+
+* Review and approve non-technical charters for DCP projects
+
+* Act as final arbiter for escalated conflicts from the DCP community
+
+*Communication*
+* Coordinate regular outreach and announcements to the DCP community and the HCA Science Leadership
```

Comment:
How is this responsibility aligned with the tasks that the communication team is taking on @jodihir @matthewspeir

Reply:
@lauraclarke I believe so. In the communications team charter, we included provisions to coordinate with the HCA Science team when communicating about scientific activities.

Reply:
I think the PMs can choose to make these communications themselves or with the help of the Communications Team. We just want to make sure we don't duplicate efforts, but I envision a lot of tasks that the Communications Team takes on as being "assigned" by the PM team so it will be clear who is doing what.

Reply:
I could either:

~~Coordinate regular outreach and announcements to the DCP community and the HCA Science Leadership~~

or

_Coordinate regular outreach and announcements to the DCP community and the HCA Science Leadership **with assistance from the Communications team**_

I'd confess that I'd been thinking of the Communications team as a "sub-committee" of DCP PM, which may be wildly inaccurate.

Reply:
I don't see it as a sub-committee of the PM team, but assisting the PMs in outreach and announcements as you wrote here @brianraymor sounds reasonable to me.

Reply:
+1 from me